core/src/test/scala/org/bykn/bosatsu/TestUtils.scala

```diff
@@ -155,18 +156,33 @@ object TestUtils {
       val normPackMap = NormalizePackageMap(infPackMap).normalizePackageMap
       (for {
         pack <- normPackMap.toMap.get(mainPack)
-        (name, rec, expr) <- pack.program.lets.lastOption
+        exprs <- pack.program.lets.traverse {case (_, rec, expr) =>
+          expr.traverse[Option, (Declaration, Normalization.NormalExpressionTag, String)]{
+            case (d, nt) => Some((d, nt, nt.ne.serialize(x => x)))
+          }
+        }
+        expr <- exprs.lastOption
+        fleft = exprs.foldLeft(0) {case (n, expr) => expr.foldLeft(n) { case (m, _) => m +1 }}
```

Comment:
I think `Foldable[List].compose[TypedExpr].size(exprs)` is the size as this.

Reply:
oh nice, I'll use that

Reply:
I couldn't figure out how to use `Foldable`'s `compose` here instead of `List`'s. So I just `map`d and `sum`d

Reply:
I realized I don't want to sum/fold/compose the lists. I just want to exercise the TypedExpr folding code